Cannot find module '@mbunifu/generator-library'

```
» npx @mbunifu/generator@0.0.2                                                                                                                             1 ↵
npx: installed 104 in 8.339s
Cannot find module '@mbunifu/generator-library'
```

--- a/index.js
+++ b/index.js
@@ -1,4 +1,4 @@
-const { Generator } = require('@mbunifu/generator-library');
+const { Generator } = require('@mbunifu/library');
 
 module.exports = class GeneratorGenerator extends Generator {
   constructor(options) {

